package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "systemml-website",
-  "version": "0.1.0",
+  "name": "systemds-website",
```

Comment:
Comment 2: This is out of scope of this PR. These changes will be taken up in a different PR.
same with `package.json` file.